flixel/input/gamepad/ButtonID.hx

```diff
@@ -0,0 +1,57 @@
+package flixel.input.gamepad;
+
+import flixel.system.macros.FlxMacroUtil;
+
+/**
+ * Maps enum values and strings to integer keycodes.
+ */
+@:enum
+abstract ButtonID(Int) from Int to Int
+{
+	public static var fromStringMap(default, null):Map<String, ButtonID>
+		= FlxMacroUtil.buildMap("flixel.input.gamepad.ButtonID");
+		
+	public static var toStringMap(default, null):Map<ButtonID, String>
+		= FlxMacroUtil.buildMap("flixel.input.gamepad.ButtonID", true);
+	
+	// Button Indices
+	var ANY               = -2;		
+	var NONE              = -1;		
+	var A                 =  0;		//BOTTOM face button
```

Comment:
If you're gonna add comments for these, use `/** */`-style comments so it shows up in completion.